lib/switchingOnOff.js

```diff
@@ -455,13 +455,11 @@ async function GroupPowerCleaningLightOnOff(adapter, Group, OnOff) {
 
 		if (OnOff) {
 			if (LightGroups[Group].power) {
-				await Promise.all([SetBrightness(adapter, Group, 100), SetCt(adapter, Group, 6500)]);
+				await Promise.all([SetBrightness(adapter, Group, 100), SetCt(adapter, Group, 6500), SimpleGroupPowerOnOff(adapter, Group, true)]);
 				LightGroups[Group].lastPower = true;
 			} else {
-				adapter.LightGroups[Group] = {
-					power: true,
-					lastPower: false,
-				};
+				adapter.LightGroups[Group].power = true;
```

Comment:
Das ist der einheitliche Fehler an der Sache! 👍 Hier wurde der gesamte Memory der Gruppe gelöscht! Dies dürfte auf den "FastFix" oben unnötig machen.

Reply:
Leider nicht ganz, hat mich aber auf den richtigen weg gebracht 👍 In Zukunft fallen die "ramp" Datenpunkte aber eh weg oder?

Reply:
ne, die rampUp/Down fallen nicht weg. Wieso auch?